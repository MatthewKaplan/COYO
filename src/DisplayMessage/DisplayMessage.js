import React from 'react';

export const DisplayMessage = (props) => {
  return(
    <article className="displayMessageContainer">
        <p className="emptyTypeText">{props.emptyType}</p>
    </article>
  )
}