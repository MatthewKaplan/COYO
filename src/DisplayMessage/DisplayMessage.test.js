import React from 'react';
import DisplayMessage from './DisplayMessage.js';
import { shallow } from 'enzyme';

describe('DisplayMessage', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
      <DisplayMessage />
    )
  });

  it('should match snapshot', () => {
  expect(wrapper).toMatchSnapshot();
  });

//test map functionality

});