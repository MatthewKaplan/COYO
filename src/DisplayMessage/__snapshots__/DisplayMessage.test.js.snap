// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DisplayMessage should match snapshot 1`] = `
<article
  className="displayMessageContainer"
>
  <p
    className="emptyTypeText"
  />
</article>
`;
